### PR TITLE
fixed Nonetype error in scipy.test_eigh_tridiagonal

### PR DESCRIPTION
**Pull request by humzakt**
# PR Description 

fixed Nonetype error in scipy.test_eigh_tridiagonal. Previously the test was not running properly now the test is functional.

Note: The tests are failing due to difference in values from ground truth and Ivy implementation. However since this is an experimental function and the scipy implementation directly uses the ivy backend experimental implementation so tests will automatically pass once the backend is properly working

### Socials: 

twitter.com/humzakt

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_scipy/test_linalg/test_linalg.py [hunk 1]
```diff
@@ -130,6 +130,8 @@ def test_scipy_eigh_tridiagonal(
     on_device,
     backend_fw,
 ):
+    print(all_args[-1])
```
Comment (umairjavaid) [inline ivy_tests/test_ivy/test_frontends/test_scipy/test_linalg/test_linalg.py:133]: remove print

**Review by umairjavaid — CHANGES_REQUESTED** [review]
remove print

File: ivy_tests/test_ivy/test_frontends/test_scipy/test_linalg/test_linalg.py (hunk 1 above)
Comment (humzakt) [reply ivy_tests/test_ivy/test_frontends/test_scipy/test_linalg/test_linalg.py]: removed

**Review by AnnaTz — APPROVED** [review]
Looks good, thanks!

**Pull request merged**